contracts/utils/Strings.sol

```diff
@@ -7,7 +7,7 @@ pragma solidity ^0.8.0;
  * @dev String operations.
  */
 library Strings {
-    bytes16 private constant _HEX_SYMBOLS = "0123456789abcdef";
+    bytes16 private constant _SYMBOLS = "0123456789abcdef";
```

Comment:
Does that change anything ?

If no, I would suggest to change as little as possible.

Reply:
It's private, so it can't break anything. I'm proposing unifying logic for getting decimal and hex characters, so the lookup table can be used in both cases making it no longer tied to the hex logic. It's purely cosmetic though.